docs/micro-lc/overview.md

```diff
@@ -25,33 +25,36 @@ Take a look to the introduction about micro frontend and how micro-lc works:
 `micro-lc` is composed by three main blocks: **fe-container**, **be-config** and a varying number
 of **plugins**.
 
-## Front-end container
+## Frontend container
 
-The front-end core of `micro-lc` is the **fe-container**: a lightweight launcher written in `React` and `Typescript` which is
-responsible for rendering the plugins and for providing a set of core functionalities to the application.
+The frontend core of `micro-lc` is the **fe-container**: a lightweight launcher written in `React` and `Typescript`. The **fe-container** 
+has two main responsabilities: 
+
+- to render the plugins 
+- providing the application a set of core functionalities.
 
 These functionalities are for the most part configurable through a [core configuration](core_configuration.md)
 retrieved by the launcher at runtime. The choice of consuming this configuration at runtime makes the fe-container
-**multi tenant**: multiple tenants can use the same instance of micro-lc controlling the rendered plugins and the
+**multi tenant**: multiple tenants can use the same instance of `micro-lc` controlling the rendered plugins and the
 application theme through different configurations.
 
 :::info
 The **fe-container** is also available as [Docker image](https://hub.docker.com/r/miaplatform/microlc).
 :::
 
-It follows a list of the features offered and handled by the **fe-container**.
+Below there is a list of the features offered and handled by the **fe-container**.
 
 ### Base layout
 
 The launcher provides the core navigation layout composed by a top bar and a side menu. The top bar is responsible for
-the top-level functionalities, while the list of plugins and the routing capabilities can be placed on top bar or side menu, 
-the latter is the default one.
+the top-level functionalities. The list of plugins and the routing capabilities are placed by default on the side menu, 
+but you can move them in the top bar.
 
 While the top bar is always visible after the configuration load, the menu is designed as an overlay to leave as much space as possible to the
 plugins.
 
 :::tip
-The plugins can implement their own sub-navigation with an eventual dedicated side menu.  
+The plugins can implement their own sub-navigation with a dedicated side menu.  
```

Comment:
This is not always true: the menu can be configured to be on the topbar.

Reply:
Ok, but I would avoid using `eventual`: it can be confusing, as in english it has a slightly different meaning than the italian `eventuale` (see `eventually`). 
Also, I think that this is just one possibility comes from the `can` at the beginning and by the fact that we are in a "Tip" box, so it's clear that this is a suggestion. However, we could add the following sentence at the end to dispel any doubt:

> The plugins can implement their own sub-navigation with a dedicated side menu.  Please note that the menu can be configured also on the topbar.